server/user.go

```diff
@@ -43,15 +43,33 @@ func (c *Connection) JiraAccountID() types.ID {
 }
 
 type ConnectionSettings struct {
-	Notifications bool `json:"notifications"`
+	Notifications          bool `json:"notifications"`
+	RolesForDMNotification map[string]*bool
```

Comment:
Thank you for the changes! I think we need much less code now. 

I am wondering, do you think we need to store space (using pointers here) for booleans? wouldn't you think would be better having only bool?